src/middleware/socket.js

```diff
@@ -5,7 +5,7 @@ module.exports = app => app.configure(
 	socketio((io) => {
 		io.on('connection', (socket) => {
 		// do some authorization things
-			app.channel('anonymous').join(socket.feathers);
+			// app.channel('anonymous').join(socket.feathers);
```

Comment:
can this be removed?

Reply:
der Change ist von @CordlessWool , gebe die Frage weiter :)

Reply:
Ich glaube er wollte ein channel für alle haben.